index.ts

```diff
@@ -15,49 +13,39 @@ async function run() {
     const client = new AirtopClient({
       apiKey: AIRTOP_API_KEY,
     });
+
     const createSessionResponse = await client.sessions.create({
       configuration: {
         timeoutMinutes: 5,
       },
     });
 
-    const session = createSessionResponse.data;
-    console.log('Created airtop session', session.id);
+    const sessionId = createSessionResponse.data.id;
+    console.log('Created airtop session', sessionId);
 
     if (!createSessionResponse.data.cdpWsUrl) {
       throw new Error('Unable to get cdp url');
     }
 
-    // Connect to the browser
-    const cdpUrl = createSessionResponse.data.cdpWsUrl;
-    const browser: Browser = await puppeteer.connect({
-      browserWSEndpoint: cdpUrl,
-      headers: {
-        Authorization: `Bearer ${AIRTOP_API_KEY}` || '',
-      },
-    });
-    console.log('Connected to browser');
+    // Create a new window and navigate to the URL
+    const windowResponse = await client.windows.create(
+      sessionId,
+      { url: TARGET_URL }
+    );
 
-    // Open a new page
-    const page: Page = await browser.newPage();
-    
-    // Navigate to the URL
-    console.log('Navigating to URL', TARGET_URL);
-    await page.goto(TARGET_URL);
-    const windowInfo = await client.windows.getWindowInfoForPuppeteerPage(session, page, {
+    const windowInfo = await client.windows.getWindowInfo(sessionId, windowResponse.data.windowId, {
       disableResize: true, // Prevents the browser window from being resized when loading a live view, which might impact the agent's ability to scrape or summarize content
```

Comment:
Did we decide to leave `disableResize`?  Or would we be better off removing it?

Reply:
I'm in favor of removing it